src/test/java/org/jqassistant/plugin/jee/cdi/test/InterceptorIT.java

```diff
@@ -1,37 +1,79 @@
 package org.jqassistant.plugin.jee.cdi.test;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
+import com.buschmais.jqassistant.core.report.api.model.Column;
 import com.buschmais.jqassistant.core.report.api.model.Result;
+import com.buschmais.jqassistant.core.report.api.model.Row;
+import com.buschmais.jqassistant.core.rule.api.model.Concept;
+import com.buschmais.jqassistant.plugin.java.api.model.TypeDescriptor;
 import com.buschmais.jqassistant.plugin.java.test.AbstractJavaPluginIT;
 
 import org.jqassistant.plugin.jee.cdi.test.set.beans.interceptor.CustomBinding;
 import org.jqassistant.plugin.jee.cdi.test.set.beans.interceptor.CustomInterceptor;
 import org.junit.jupiter.api.Test;
 
-import static com.buschmais.jqassistant.plugin.java.test.matcher.TypeDescriptorMatcher.typeDescriptor;
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.hasItem;
-import static org.hamcrest.MatcherAssert.assertThat;
+import static com.buschmais.jqassistant.core.report.api.model.Result.Status.SUCCESS;
+import static com.buschmais.jqassistant.plugin.java.test.assertj.TypeDescriptorCondition.typeDescriptor;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * Tests for the interceptor concepts.
  */
 class InterceptorIT extends AbstractJavaPluginIT {
 
     /**
-     * Verifies the concept "cdi:Interceptor".
-     *
-     * @throws java.io.IOException
-     *             If the test fails.
+     * Verifies the concept "interceptor:Interceptor".
      */
     @Test
     void interceptor() throws Exception {
         scanClasses(CustomInterceptor.class);
-        assertThat(applyConcept("interceptor:Interceptor").getStatus(), equalTo(Result.Status.SUCCESS));
+        final Result<Concept> conceptResult = applyConcept("interceptor:Interceptor");
+        assertThat(conceptResult.getStatus()).isEqualTo(SUCCESS);
+        store.beginTransaction();
+        final List<TypeDescriptor> conceptResultTypes = conceptResult.getRows().stream()
+                .map(Row::getColumns)
+                .map(columns -> columns.get("Interceptor"))
+                .map(Column::getValue)
+                .map(TypeDescriptor.class::cast)
+                .collect(Collectors.toList());
+        assertThat(conceptResultTypes).hasSize(1);
+        assertThat(conceptResultTypes).haveExactly(1, typeDescriptor(CustomInterceptor.class));
+
+        final List<TypeDescriptor> interceptorTypes =
```

Comment:
Would it be easier to have only one query matching for Injectable and Interceptor at the same time?